openlibrary/templates/home/categories.html

```diff
@@ -46,46 +45,44 @@ <h2 class="home-h2">
 
     var Categories = {
         featured: {
-            'art': {'name': 'Art', 'sort': 'edition_count'},
-            'science_fiction': {'name': 'Sci-Fi', 'sort': 'edition_count'},
-            'fantasy': {'name': 'Fantasy', 'sort': 'edition_count'},
-            'biographies': {'name': 'Biographies', 'sort': 'edition_count'},
-            'recipes': {'name': 'Recipes', 'sort': 'edition_count'},
-            'romance': {'name': 'Romance', 'sort': 'edition_count'},
-            'textbooks': {'name': 'Textbooks', 'sort': 'edition_count'},
-            'children': {'name': 'Children', 'sort': 'edition_count'},
-            'history': {'name': 'History', 'sort': 'edition_count'},
-            'medicine': {'name': 'Medicine', 'sort': 'edition_count'},
-            'religion': {'name': 'Religion', 'sort': 'edition_count'},
-            'mystery_and_detective_stories': {'name': 'Mysteries', 'sort': 'edition_count'},
-            'plays': {'name': 'Plays', 'sort': 'edition_count'},
-            'music': {'name': 'Music', 'sort': 'edition_count'},
-            'science': {'name': 'Science', 'sort': 'edition_count'}
-        },
-        get: function(category) {
-            var url = '/subjects/' + category + '.json';
+            'art': {'name': 'Art'},
+            'science_fiction': {'name': 'Sci-Fi'},
+            'fantasy': {'name': 'Fantasy'},
+            'biographies': {'name': 'Biographies'},
+            'recipes': {'name': 'Recipes'},
+            'romance': {'name': 'Romance'},
+            'textbooks': {'name': 'Textbooks'},
+            'children': {'name': 'Children'},
+            'history': {'name': 'History'},
+            'medicine': {'name': 'Medicine'},
+            'religion': {'name': 'Religion'},
+            'mystery_and_detective_stories': {'name': 'Mysteries'},
+            'plays': {'name': 'Plays'},
+            'music': {'name': 'Music'},
+            'science': {'name': 'Science'}
         },
         setup: function(categories) {
             for (var key in categories) {
                 var category = categories[key];
                 \$('.categoryCarousel').append(Subject.render(category, key));
             }
-            for (var cat in categories) {
-                Subject.count(cat, function(subject, count) {
-                    var cnt = count.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
-                    \$('[name="' + subject + '"]').text(cnt + ' books');
-                });
-            }
+
+            Subject.get_featured(function(results) {
+                for (var subject_key in results) {
+                    var subject = results[subject_key];
+                    console.log(subject);
+                    var cnt = (subject.work_count || 0).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
Can you add a comment explaining the regexp?

Also, I suggest "count" over "cnt", an unnecessary and obfuscating abbreviation.